lib/sequence.js

```diff
@@ -53,6 +53,18 @@ export default class Sequence {
       });
   }
 
+  static async getLatestMigration(fs, dirPath) {
+    const sequence = await this.readMigrations(fs, dirPath);
```

Comment:
So this is pretty simple - `readMigrations` at its base level will find _every_ migration that exists for a schema in their files and store it in an array. It orders it from lowest to highest.